profiles/coreos/targets/generic/make.defaults

```diff
@@ -38,6 +38,7 @@ INSTALL_MASK="${INSTALL_MASK}
   /etc/sudoers
   /etc/wgetrc
   /etc/xinetd.d
+  /etc/vim/vimrc
```

Comment:
Possibly add /usr/share/vim/vim82/doc/gentoo-syntax.txt there too.

Reply:
And `/usr/share/vim/vim82/tools/vim132`, maybe with a comment about `gentoo-syntax` being a broken link, and vim132 using unsupported shell (`csh`).